sound/soc/intel/boards/sof_da7219_max98373.c

```diff
@@ -247,6 +271,17 @@ SND_SOC_DAILINK_DEF(idisp3_codec,
 SND_SOC_DAILINK_DEF(platform, /* subject to be overridden during probe */
 	DAILINK_COMP_ARRAY(COMP_PLATFORM("0000:00:1f.3")));
 
+static struct snd_soc_dai_link_component ssp1_codec_components[] = {
+	{
+		.name = MAXIM_DEV0_NAME,
+		.dai_name = MAX98373_CODEC_DAI,
+	},
+	{
+		.name = MAXIM_DEV1_NAME,
+		.dai_name = MAX98373_CODEC_DAI,
+	},
+};
+
```

Comment:
don't we have a way to use the DAILINK_ macros? It's a bit ugly to hard-code when everything else uses the macros, no?

Reply:
I did not find a better way last time, let me double check, thanks!!

Reply:
SND_SOC_DAILINK_DEF(ssp1_codec,
	DAILINK_COMP_ARRAY(
	/* Left */	COMP_CODEC(MAXIM_DEV0_NAME, MAX98373_CODEC_DAI),
	/* Right */	COMP_CODEC(MAXIM_DEV1_NAME, MAX98373_CODEC_DAI)));

?